cernopendata/modules/fixtures/data/docs/cms-mc-production-overview/cms-mc-production-overview.md

```diff
@@ -1,3 +1,5 @@
+This guide gives a brief overview of the main steps in the production chain of simulated data, and its main objective is to explain how to find the parameters used in event generation for datasets that are available as public data. These parameters are displayed in the simulated dataset records whenever possible, but in some cases they are well hidden in different configuration files, or only available with a command line script reading them from a data file directly. If you are interested in producing new simulated data, read [the guide for event production](../docs/cms-guide-event-production).
```

Comment:
In the link target, instead of `.../docs/xxx` please use `/docs/xxx`.

Reply:
Thanks, done.